EIPS/eip-5757.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
EIPW says you are missing a Security Considerations section.

So that this can be merged faster I suggest you add one ;)

Reply:
I'm going to add this EIP to the unchecked list.